pages/pipelines/glossary.md

```diff
@@ -0,0 +1,72 @@
+# Pipelines glossary
+
+The following terms describe key concepts to help you use Pipelines.
+
+{:toc}
+
+## Agent
+
+An agent is a small, reliable, and cross-platform build runner that connects your infrastructure to Buildkite. It polls Buildkite for work, runs jobs, and reports results. You can install agents on local machines, cloud servers, or other remote machines. You need at least one agent to run builds.
+
+To learn more, see the [Agent overview](/docs/agent/v3).
+
+## Artifact
+
+An artifact is a file generated during a build. You can keep artifacts in a Buildkite-managed storage service or a third-party cloud storage service like Amazon S3, Google Cloud Storage, or Artifactory. Common uses include storing assets like logs and reports, or passing files between steps.
+
+To learn more, see [Build artifacts](/docs/pipelines/artifacts).
+
+## Build
+
+A build is a single run of a pipeline. You can trigger a build in various ways, including through the UI, API, as the result of a webhook, or on a schedule.
```

Comment:
@mbelton-buildkite you can also use a trigger step to create a build: https://buildkite.com/docs/pipelines/trigger-step

Reply:
I like the way that hints at the complexity you can achieve with Buildkite. I'll include it, thanks! 😊